src/commands/schema/generate.ts

```diff
@@ -84,11 +108,33 @@ export class SchemaGenerator {
 
   private getAllHookFiles(): string[] {
     // eslint-disable-next-line unicorn/no-array-reduce
-    const hookFiles = Object.values(this.base.config.pjson.oclif?.hooks ?? {}).reduce((x: string[], y: string | string[]) => {
-      return Array.isArray(y) ? [...x, ...y] : [...x, y]
-    }, [] as string[])
-    const {rootDir, outDir} = this.getDirs()
-    return (hookFiles as string[]).map(h => `${path.resolve(h)}.ts`.replace(outDir, rootDir))
+    const hookFiles = Object.values(this.base.config.pjson.oclif?.hooks ?? {}).reduce(
+      (x: string[], y: string | string[]) => (Array.isArray(y) ? [...x, ...y] : [...x, y]),
+      [] as string[],
+    )
+    const {outDir, rootDir} = this.getDirs()
+    return (hookFiles as string[]).map((h) => `${path.resolve(h)}.ts`.replace(outDir, rootDir))
```

Comment:
```suggestion
    return (hookFiles).map((h) => `${path.resolve(h)}.ts`.replace(outDir, rootDir))
```

Reply:
seems like the types have been strengthened, but the usages are still using `as ...`